src/rest.rs

```diff
@@ -1128,6 +1128,26 @@ fn handle_request(
 
             json_response(prepare_txs(txs, query, config), TTL_SHORT)
         }
+        (&Method::POST, Some(&"mempool"), Some(&"txs"), None, None, None) => {
+            let txid_strings: Vec<String> =
+                serde_json::from_slice(&body).map_err(|err| HttpError::from(err.to_string()))?;
+
+            match txid_strings
+                .into_iter()
+                .map(|txid| Txid::from_hex(&txid))
+                .collect::<Result<Vec<Txid>, HashHexError>>()
```

Comment:
You can just use `_` for the error to avoid the extra import. (Since we just call to_string anyways we don't really care about the error.)